Source/cmake/OptionsWin.cmake

```diff
@@ -205,7 +205,10 @@ if (USE_LCMS)
 endif ()
 
 if (USE_JPEGXL)
-    find_package(JPEGXL REQUIRED)
+    find_package(JPEGXL 0.7.0)
```

Comment:
Why not using `REQUIRED` here?

Reply:
This follows along with what WPE and GTK do where they do a find and if it’s not there give a message. In this case it can be user specified so they prefer giving a more explicit message on why it failed